### ISSUE TITLE
Can't locate Test/Differences.pm in @INC

### ISSUE THREAD

**Issue by eserte** [zonemaster/zonemaster-ldns#183]
t/packet.t and t/rr.t fail if Test::Differences is not installed:
```
Can't locate Test/Differences.pm in @INC (you may need to install the Test::Differences module) (@INC contains: ... .) at t/packet.t line 6.
BEGIN failed--compilation aborted at t/packet.t line 6.
t/packet.t ......... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
Can't locate Test/Differences.pm in @INC (you may need to install the Test::Differences module) (@INC contains: ... .) at t/rr.t line 6.
BEGIN failed--compilation aborted at t/rr.t line 6.
t/rr.t ............. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by matsduf** [comment]
Could you describe the steps that you took before you got this message? I can see that the README is not fully updated. If you want to install Zonemaster, please follow the steps in https://github.com/zonemaster/zonemaster/tree/master/docs/public/installation#readme

**Comment by eserte** [comment]
I am running multiple CPAN smokers, using CPAN.pm. So if the prerequisites are declared correctly, then everything should be installed automatically.

**Comment by eserte** [comment]
The detailed fail reports can be found on http://matrix.cpantesters.org/?dist=Zonemaster-LDNS%204.0.0;reports=1
The two fail reports from freebsd are from my smokers.
(The other fail reports from openbsd seem to be wrongly flagged, probably these should be "unknown" or "na" reports due to missing system requirements).

**Comment by eserte** [comment]
The following Dockerfile reproduces the problem (note that I am not using docker within the smokers, but docker is a good tool to start from a clean system with as few packages as possible installed) (note that the huge list of perl packages in the Dockerfile is not strictly necessary to reproduce the problem, this is just here to speed up the installation):
```
# docker build -t perl-test . && docker run --rm perl-test

FROM debian:bookworm
RUN apt-get -y update
RUN apt-get -y install --no-install-recommends perl perl-modules make
RUN apt-get -y install --no-install-recommends gcc
RUN apt-get -y install --no-install-recommends libc6-dev

## just to speed up installation of CPAN modules
RUN apt-get -y install --no-install-recommends libarchive-zip-perl libb-hooks-op-annotation-perl libcanary-stability-perl libcapture-tiny-perl libclone-perl libcommon-sense-perl libcpanel-json-xs-perl libdevel-checklib-perl libdevel-hide-perl libencode-locale-perl libextutils-config-perl libextutils-depends-perl libextutils-helpers-perl libextutils-installpaths-perl libffi-checklib-perl libfile-chdir-perl libfile-homedir-perl libfile-listing-perl libfile-remove-perl libfile-which-perl libhtml-parser-perl libhtml-tagset-perl libhttp-cookiejar-perl libhttp-cookies-perl libhttp-daemon-perl libhttp-date-perl libhttp-message-perl libhttp-negotiate-perl libio-html-perl libio-socket-ssl-perl libipc-run3-perl libjson-maybexs-perl libjson-perl libjson-xs-perl liblwp-mediatypes-perl libmime-charset-perl libmodule-build-perl libmodule-build-tiny-perl libmodule-install-perl libmodule-install-xsutil-perl libmodule-scandeps-perl libnet-http-perl libnet-ssleay-perl libpar-dist-perl libpath-tiny-perl libpkgconfig-perl libsort-versions-perl libterm-size-any-perl libterm-size-perl-perl libtest-deep-perl libtest-failwarnings-perl libtest-fatal-perl libtest-leaktrace-perl libtest-mockrandom-perl libtest-needs-perl libtest-number-delta-perl libtest-requires-perl libtest-requiresinternet-perl libtest-warnings-perl libtest2-suite-perl libtimedate-perl libtry-tiny-perl libtypes-serialiser-perl libunicode-linebreak-perl libunicode-utf8-perl liburi-perl libwww-perl libwww-robotrules-perl libyaml-tiny-perl

RUN apt-get -y install --no-install-recommends libssl-dev
RUN apt-get -y install --no-install-recommends libidn2-dev

## the missing test requirement
#RUN apt-get -y install --no-install-recommends libtest-differences-perl

CMD cpan -t Zonemaster::LDNS
```

**Comment by matsduf** [comment]
@eserte, thank you for the information. #185 should resolve this issue.

**Comment by matsduf** [comment]
Resolved by #185
